src/components/inputFields/InputFieldCustomIcon.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import {
+    responsiveWidth,
+    responsiveFontSize
+} from "react-native-responsive-dimensions";
+import { HelperText, TextInput } from 'react-native-paper';
+import Icon from 'react-native-vector-icons/Feather';
+import { InputFieldCustomIconInterface } from '../../interface/inputFields/InputFieldCustomIconInterface';
+
+export const InputFieldCustomIcon : React.FC<InputFieldCustomIconInterface & { onChangeText: (text: string) => void }> = ({
```

Comment:
Change InputFieldCustomIcon to InputFieldWithIcon

Reply:
done.